ios/testmanagerd/xctestrunutils.go

```diff
@@ -25,8 +24,7 @@ import (
 
 // xCTestRunData represents the structure of an .xctestrun file
 type xCTestRunData struct {
-	TestConfig        schemeData        `plist:"-"`
-	XCTestRunMetadata xCTestRunMetadata `plist:"__xctestrun_metadata__"`
+	TestConfig schemeData `plist:"-"`
```

Comment:
This is not really used for plist parsing anymore, isn't it? And since `schemeData` is the only field left, can't we just get rid of this struct and use `TestConfig` instead?